docs/update.md

```diff
@@ -0,0 +1,37 @@
+# Updating Groups
+
+Inago includes an `update` command, which allows updating
+a group on the cluster.
+
+__Note__: The `update` command only works on sliced groups.
+
+## Usage
+
+Updating a group includes stopping old slices and starting new ones.
+The behavior of the update process is determined by two flags: `--min-alive` and `--max-growth`.
+
+### min-alive
+The `--min-alive` flag defines the minimum number of slices,
+that must be running at any time during the update process.
+
+### max-growth
+The `--max-growth` flag sets the upper limit on how many additional 
+slices may be temporarily started during the update process.
```

Comment:
I'd remove "temporarily" here - the additional slices may be the ones that we keep running, in which case they're not temporary.